articles/storage/blobs/lifecycle-management-policy-configure.md

```diff
@@ -228,6 +228,9 @@ A lifecycle management policy must be read or written in full. Partial updates a
 > [!NOTE]
 > The delete action of a lifecycle management policy won't work with any blob in an immutable container. With an immutable policy, objects can be created and read, but not modified or deleted. For more information, see [Store business-critical blob data with immutable storage](./immutable-storage-overview.md).
 
+> [!NOTE]
+> Kindly Note that while configuring the life cycle management policy if you add blob type as append blob, in that case for applying rule on base blob last accessed days option would not be available as it is not supported for append blob.
```

Comment:
@jainsourabh123 Can we add this note after the opening paragraph of the "Optionally enable access time tracking" section. The list of pink notes is getting pretty large in that section. The only way that last access tracking can be used in a rule is if it is enabled so folks will have to look at the "Optionally enable access time tracking" paragraph to enable it.  The note could probably be shorted up a bit to "Append blobs don't support last access tracking". I assume that this is true based on the fact that rules can't use that condition for append blobs. Please let me know if that is an incorrect assumption.

Reply:
Awesome. Thanks @jainsourabh123 for working with me on this. I updated the text in this PR into your branch. Can you pls merge? - [https://github.com/jainsourabh123/azure-docs/pull/3](https://github.com/jainsourabh123/azure-docs/pull/3)

Thx!

Reply:
Yikes. Also forgot to remove the other text. Here's a PR for that - [https://github.com/jainsourabh123/azure-docs/pull/4](https://github.com/jainsourabh123/azure-docs/pull/4). If you wouldn't mind merging both of these, then we can sign off on this PR. Thx!